### PR TITLE
StatusCard on mobile screens hides the device on the map

### PR DESCRIPTION
**Pull request by jcardus**
before:
traccar fleetmap io_(iPhone SE) (1)

after:
localhost_3000_(iPhone SE)

### CODE REVIEW

File: modern/src/common/components/StatusCard.js [hunk 1]
```diff
@@ -57,6 +57,8 @@ const useStyles = makeStyles((theme) => ({
   content: {
     paddingTop: theme.spacing(1),
     paddingBottom: theme.spacing(1),
+    'max-height': '40vh',
+    overflow: 'hidden',
```
Comment (tananaev) [inline modern/src/common/components/StatusCard.js:61]: Why are we hiding overflow?

Comment (jcardus) [reply modern/src/common/components/StatusCard.js:61]: because it overlaps with the actions

Comment (jcardus) [reply modern/src/common/components/StatusCard.js:61]: without overflow: hidden

localhost_3000_(iPhone SE) (3)

Comment (tananaev) [reply modern/src/common/components/StatusCard.js:61]: Screenshot please. And also wouldn't it mean that it's not the content is not accessible?

Comment (jcardus) [reply modern/src/common/components/StatusCard.js:61]: yes, the content overflow content will not accesible

Comment (tananaev) [reply modern/src/common/components/StatusCard.js:61]: That's not an acceptable solution then. It should become scrollable if you want to limit the height.

Comment (jcardus) [reply modern/src/common/components/StatusCard.js]: Changed to overflow: scroll
localhost_3000_(iPhone SE) (4)

File: modern/src/common/components/StatusCard.js [hunk 2]
```diff
@@ -57,6 +57,8 @@ const useStyles = makeStyles((theme) => ({
   content: {
     paddingTop: theme.spacing(1),
     paddingBottom: theme.spacing(1),
+    'max-height': '40vh',
```
Comment (tananaev) [inline modern/src/common/components/StatusCard.js:60]: You have to use camelcase name here. And also the constant should be in the same place where we put other similar constants. It's part of the theme.

**Comment by jcardus** [comment]
please check

**Comment by tananaev** [comment]
Merged, thanks.

**Comment by tananaev** [comment]
@jcardus is looks like the card is broken by this change. Tested on Chrome on Mac. Can you please fix.

Screenshot 2023-08-19 at 2 05 04 PM

**Comment by jcardus** [comment]
Sorry @tananaev, I just saw this now. Is this still an issue?

**Comment by tananaev** [comment]
I forgot. Maybe I already fixed it?

**Comment by jcardus** [comment]
I can't reproduce it.

**Pull request merged**